Tipstaff/DAL/TipstaffInitializer.cs

```diff
@@ -404,7 +404,7 @@ protected override void Seed(TipstaffDB context)
                 new Division{Detail="Chancery", Prefix="CH", active=true},
                 new Division{Detail="Family", Prefix="FAM", active=true},
                 new Division{Detail="Insolvency", Prefix="IN", active=true},
-                new Division{Detail="Queen's Bench", Prefix="QB", active=true}
```

Comment:
Shouldn't the QB entry just be hanged to active=false (this will automatically update the database I think)